### PR TITLE
CHROMEOS build-configs-chromeos.yaml: Add 5.15 LTS stable tree

### PR DESCRIPTION
**Pull request by nuclearcat**
Add a chromeos-stable_5.15 build configuration based on v5.15 LTS.

Signed-off-by: Denys Fedoryshchenko <denys.f@collabora.com>

### CODE REVIEW

File: config/core/build-configs-chromeos.yaml [hunk 1]
```diff
@@ -57,6 +57,11 @@ build_configs:
     branch: 'linux-5.10.y'
     variants: *chromeos-variants
 
+  chromeos-stable_5.15:
```
Comment (gctucker) [inline config/core/build-configs-chromeos.yaml:60]: I guess we probably should also rename `chromeos-stable` to `chromeos-stable_5.10`

Comment (nuclearcat) [reply config/core/build-configs-chromeos.yaml:60]: Done

**Comment by nuclearcat** [comment]
Rebased

**Comment by gctucker** [comment]
I've also updated the monitor job parameters accordingly:
```json
{
    "CONFIG_LIST": "chromeos-stable_5.10 kernelci-stable_5.15 kernelci_chromeos-stable"
}
```
This should be added to `kernelci-deploy` once the current PR is merged.

**Comment by nuclearcat** [comment]
Rebased

**Comment by gctucker** [comment]
@nuclearcat Please rebase once more.

**Comment by nuclearcat** [comment]
Done

**Comment by gctucker** [comment]
Tested OK: https://chromeos.kernelci.org/job/stable/branch/linux-5.15.y/

**Pull request merged**